caffe2/operators/conv_op_cudnn.cc

```diff
@@ -153,11 +153,11 @@ class CudnnConvOpBase : public ConvPoolOpBase<CUDAContext> {
               desc_,
               cudnnTypeWrapper<T>::type,
               N,
-#if CUDNN_VERSION_MIN(7,0,0)
+//#if CUDNN_VERSION_MIN(7,0,0)
```

Comment:
Why commented these lines for cudnn7?

Reply:
MSVC does not allow macro definition within functions

I think following would be better write in separate form
<      case StorageOrder::NCHW:
        if (size == 4) 
		{
		#if CUDNN_VERSION_MIN(7,0,0)
		CUDNN_ENFORCE(cudnnSetTensor4dDescriptorEx(
			desc_,
			cudnnTypeWrapper<T>::type,
			N,
			C,
			H,
			W,
			C * H * W,
			H * W,
			W,
			1));
		#else
			CUDNN_ENFORCE(cudnnSetTensor4dDescriptorEx(
				desc_,
				cudnnTypeWrapper<T>::type,
				N,
              C / group_,
              H,
              W,
              C * H * W,
              H * W,
              W,
              1));
		#endif
        } >